docs/index.rst

```diff
@@ -12,12 +12,124 @@ COSMOS Academy
    :alt: cosmos-academy-logo
    :align: center
 
+Welcome to the Cosmos Academy! This page will orient you around the Tendermint/Cosmos tech stack. It is roughly organized from beginner => expert with higher priority to content most relevant to many users.
 
-Contents
+Introduction
+============
+
+New here? Walk through some introductory content to get up to speed.
+
+.. toctree::
+   :maxdepth: 2
+
+   content/introduction.rst
+
+Testnets
+========
+
+Either create you own (a good first step) or join an existing testnet.
+
+.. toctree::
+   :maxdepth: 2
+
+   content/testnets.rst
+
+App Development
+===============
+
+Apps can be built either using the SDK or in any language for which an ABCI server exists.
```

Comment:
Discover the various development frameworks available to build your blockchain application.